gems/aws-sdk-s3/lib/aws-sdk-s3/express_credentials_provider.rb

```diff
@@ -18,21 +20,35 @@ class ExpressCredentialsProvider
       #   credentials are refreshed.
       def initialize(options = {})
         @client = options.delete(:client)
-        @caching = options.delete(:caching) || true
+        @caching = options.delete(:caching) != false
```

Comment:
good catch

Reply:
I only noticed because you had a different comment ...